### ISSUE TITLE
Question about tensorrt model

### ISSUE THREAD

**Issue by hahapt** [megvii-research/MOTRv2#10]
Can MOTRv2 model be converted to tensorrt?
Can you share the pth trained with MOT20?

**Comment by vaesl** [comment]
Yes, the MOTRv2 model can be converted to tensorrt. In our practice, it can run at 14ms with int8 bit. In the future, we may release the tensorrt version of it. As for the MOT20, we have not conducted the exps on this dataset.

**Comment by hahapt** [comment]
> Yes, the MOTRv2 model can be converted to tensorrt. In our practice, it can run at 14ms with int8 bit. In the future, we may release the tensorrt version of it. As for the MOT20, we have not conducted the exps on this dataset.

Sorry, I mean the MOT17 mentioned in paper.

**Comment by vaesl** [comment]
Since the paper is still under review, I think it will also be released in the near future based on the submission result. Currently, we simply released the DanceTrack one.

**Comment by hahapt** [comment]
> Since the paper is still under review, I think it will also be released in the near future based on the submission result. Currently, we simply released the DanceTrack one.

OK fine. So how to convert the MOTRv2 model to tensorrt? I try to convert it by first convert to onnx model, but I'm confused about the input dict. Can the MOTRv2 model be converted to tensorrt directly by trtexec?

**Comment by vaesl** [comment]
We first convert the MOTRv2 to onnx model and then convert the onnx to trt one. As for the input dict question, you may rethink the forward process and refer to https://github.com/talebolano/Tensorrt-Deformable-Detr. One more detail is that the DETR and QIM parts are converted to the trt models, respectively.

**Comment by HassanAliAnsari** [comment]
Were you able to convert it to tensorrt?

**Comment by danjump** [comment]
> We first convert the MOTRv2 to onnx model and then convert the onnx to trt one. As for the input dict question, you may rethink the forward process and refer to https://github.com/talebolano/Tensorrt-Deformable-Detr. One more detail is that the DETR and QIM parts are converted to the trt models, respectively.

Can you give any more guidance of the process to convert MOTRv2 to onnx? Do you have any example code from this codebase (or how to adapt this model code to follow a process similar to the link you reference?)

**Comment by lucy3589** [comment]
@vaesl 您好，可以详细说一下，如何分别转onnx吗？这两个部分怎么拆开转呢？
